### PR TITLE
GUAC-1512: Add absolute connection limit properties for MySQL/PostgreSQL.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/glyptodon/guacamole/auth/jdbc/tunnel/RestrictedGuacamoleTunnelService.java [hunk 1]
```diff
@@ -113,10 +128,54 @@
 
     }
 
+    /**
+     * Attempts to increment the given AtomicInteger without exceeding the
+     * specified maximum value. If the AtomicInteger cannot be incremented
+     * without exceeding the maximum, false is returned.
+     *
+     * @param counter
+     *     The AtomicInteger to attempt increment.
```
Comment (jmuehlner) [inline extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/glyptodon/guacamole/auth/jdbc/tunnel/RestrictedGuacamoleTunnelService.java]: Did you mean "attempt to increment"?

File: extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/java/net/sourceforge/guacamole/net/auth/mysql/MySQLEnvironment.java [hunk 2]
```diff
@@ -52,6 +52,12 @@
     private static final int DEFAULT_PORT = 3306;
 
     /**
+     * The default value for the maximum number of connections to be
+     * allowed to the Guacamole server overall.
+     */
+    private final int ABSOLUTE_MAX_CONNECTIONS = 0;
```
Comment (jmuehlner) [inline extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-mysql/src/main/java/net/sourceforge/guacamole/net/auth/mysql/MySQLEnvironment.java]: Why is this not DEFAULT_ like the others above?

File: extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/glyptodon/guacamole/auth/postgresql/PostgreSQLEnvironment.java [hunk 3]
```diff
@@ -51,6 +51,12 @@
     private static final int DEFAULT_PORT = 5432;
 
     /**
+     * The default value for the maximum number of connections to be
+     * allowed to the Guacamole server overall.
+     */
+    private final int ABSOLUTE_MAX_CONNECTIONS = 0;
```
Comment (jmuehlner) [inline extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-postgresql/src/main/java/org/glyptodon/guacamole/auth/postgresql/PostgreSQLEnvironment.java]: Same question here

File: extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/glyptodon/guacamole/auth/jdbc/tunnel/RestrictedGuacamoleTunnelService.java (hunk 1 above)
Comment (mike-jumper) [inline extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/glyptodon/guacamole/auth/jdbc/tunnel/RestrictedGuacamoleTunnelService.java]: Yep. I'll fix that.

**Pull request merged**